scene/gui/control.cpp

```diff
@@ -1620,6 +1604,21 @@ void Control::update_minimum_size() {
 		return;
 	}
 
+	Size2 minsize = get_combined_minimum_size();
```

Comment:
Ensuring that `data.minimum_size_valid` is changed from `false` to `true` only happens here. 

Previously switched to `false` in `update_minimum_size()`, `get_combined_minimum_size()` may have been called multiple times before getting here.